### ISSUE TITLE
[Feature] Show snatched on schedule screen

### ISSUE THREAD

**Issue by chris-gh** [SickRage/SickRage#3600]
When an episode has been snatched it disappears from the scheduled screen.

When that download fails, the only way to attempt to find the failure is to look under history and try and find episodes that don't have a future download status.

The scheduled screen should continue to show snached episodes until it has reached a state of downloaded. This should either be shown under "missing" or a maybe a new status.

This also should maybe have a timeout so that older snatched items are only visible for a number of days (perhaps 7 days, or a user configurable value).

**Comment by TheRealNoob** [comment]
How do I disable this feature?  Since I don't allow Sickrage to do my post-processing (I have other processes for that) my shows never make it to the "downloaded" state.

Please tell me this can be disabled....

**Comment by chris-gh** [comment]
Hi, I'll look at adding a switch to turn it on/off.

For the moment to clear the screen you can reduce the number of days under config / general / interface and change "Missed episodes range" to a low number (1 or 2).

**Comment by OmgImAlexis** [comment]
@TheRealNoob why not let sickrage just see the show directory as soon as it sees the new files it should mark it as watched.

**Comment by TheRealNoob** [comment]
@OmgImAlexis
Because all of my shows are on a network drive and I've had trouble getting sickrage to accept that.

**Comment by chris-gh** [comment]
You can use a symbolic link to get it working.

From memory it's (from a run-as-admin command prompt):
```
mklink   /D    C:\TV   \\192.168.0.100\TVShows
```
Then make sure your sickrage service is logged in as a user that has access to that drive (ie. your login user).

Map sickrage to point to the local folder, and then everything works fine. It's how I have mine set up with the files being stored on my NAS and I have no issues at all with it.

**Comment by TheRealNoob** [comment]
TL;DR Got it working :)

I know I tried to do this like a year ago and I fought with it for so long....now a year later it takes me 30 minutes.  Sometimes I just don't know why I try lol.  But I got it figured out.  I already had the service running under the correct user.  My issue was that the Symlink wouldn't point to the \\UNC path because of a username/password error.  Fortunately my Windows username already matched that of the NAS login username, so all I had to do was change my Windows password and recreate the Symlink and voila!
